packages/plugin-ext/src/common/network.ts

```diff
@@ -0,0 +1,63 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+// copied from https://github.com/Microsoft/vscode/blob/52306b406d9231a0667c758e536bd177e5570180/src/vs/base/common/network.ts
```

Comment:
as far as I understand, we still need CQs for copied code

@marcdumais-work

Reply:
I've removed it not to depend on a CQ